pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -549,16 +549,27 @@ func getMockDID() *did.Doc {
 		Service: []did.Service{
 			{
```

Comment:
TODO: What shall we do with the ;did-communication suffix?
Example from the Indy version: https://github.com/hyperledger/aries-rfcs/issues/317#issuecomment-556257803

Reply:
Ok, created issue #842 to address this.